test/index.js

```diff
@@ -353,6 +355,7 @@ test('Category', function() {
   eq(Z.Category.test([]), false);
   eq(Z.Category.test({}), false);
   eq(Z.Category.test(Math.abs), true);
+  eq(Z.Category.test({constructor: function NotCategory() {}}), false);
```

Comment:
Would it be appropriate to use [`Useless`][1] here instead?


[1]: https://github.com/sanctuary-js/sanctuary-useless